### PR TITLE
PCT: Add Holy Hammer combo feature

### PR DESCRIPTION
**Pull request by Propagandist1**
- Adds a feature to replace Holy in White with Hammer Brush and its combo chain when they are usable.  
- Named Holy Hammer Combo.
- Falls under the Holy Comet Combo parent group. 
- Does not have priority over the Rainbow Holy Combo.
- Edited the Rainbow Holy Combo description to clarify it has priority over this Holy Hammer Combo.
- My first pull request, please forgive any errors!

Addresses feature request #444

### CODE REVIEW

**Comment by MKhayle** [comment]
thanks a lot! I'll check that out before next release

File: XIVComboExpanded/Combos/PCT.cs [hunk 1]
```diff
@@ -286,6 +286,11 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     return PCT.RainbowDrip;
                 }
 
+                if (IsEnabled(CustomComboPreset.PictomancerHolyHammerCombo) && HasEffect(PCT.Buffs.HammerReady))
+                {
+                    return PCT.HammerStamp;
```
Comment (kaedys) [inline XIVComboExpanded/Combos/PCT.cs:291]: @MKhayle before you release this, this line will need to be an `OriginalHook()` call to properly use the entire hammer chain, I believe.

**Comment by Propagandist1** [comment]
Thank you, kaedys!  I think I saw OriginalHook() on some other functions, but wasn't sure entirely what it did.  I pretty much just adapted my edit from the line above on PCT.RainbowDrip.  I appreciate you looking at this and checking the code!

Looking forward to the next release!

**Comment by kaedys** [comment]
`OriginalHook()` takes an action ID, in this case Hammer Stamp, and replaces it with the ID of whatever action would replace the original on the hotbar.  So when you use Hammer Stamp, then Hammer Stamp turns into Hammer Brush, and then into Polishing Hammer.  Each of these has different action IDs.  If you return Hammer Stamp directly, then that button will cease working after the first of the hammer attacks, because it'll still be trying to use the first ability in the combo, which is no longer usable.  `OriginalHook()` fixes that by adjusting the ID to whatever would be replacing Hammer Stamp on the hotbar.  Not that this only counts vanilla game replacements, action replacements due to combos in this mod are not included in that.

**Comment by Propagandist1** [comment]
That makes a lot of sense.  I guess that's why the PictomancerWeaponHammerCombo "Hammer Time" uses OriginalHook(PCT.HammerStamp).  Oh man it's been a hot minute since programming in undergrad... 

Thank you for explaining this to me!

**Comment by kaedys** [comment]
Naw, the PR is already merged, probably easier for MKhayle to just make the tweak directly before issuing the release commit.

**Comment by MKhayle** [comment]
yes, will do today

**Pull request merged**